patroni/ctl.py

```diff
@@ -111,7 +111,7 @@ def post_patroni(member, endpoint, content, headers={'Content-Type': 'applicatio
     url = urlparse(member.api_url)
     logging.debug(url)
     return requests.post('{}://{}/{}'.format(url.scheme, url.netloc, endpoint), headers=headers,
-                         data=json.dumps(content), timeout=5)
+                         data=json.dumps(content), timeout=30)
```

Comment:
I would make timeout a little bit bigger then 30 seconds. Corresponding code in the `Api` which does manual failover will wait 30 seconds for a new leader in configuration store and then generate a response. This means that it could take more then 30 seconds to process request and your code might experience timeout...